src/JustSaying/Messaging/IMessageBatchPublisher.cs

```diff
@@ -5,7 +5,7 @@ namespace JustSaying.Messaging;
 
 public interface IMessageBatchPublisher : IInterrogable, IStartable
 {
-    Task PublishAsync(IEnumerable<Message> messages, CancellationToken cancellationToken);
-    Task PublishAsync(IEnumerable<Message> messages, PublishBatchMetadata metadata, CancellationToken cancellationToken);
+    Task PublishAsync(IEnumerable<Message> messages, CancellationToken cancellationToken = default);
+    Task PublishAsync(IEnumerable<Message> messages, PublishBatchMetadata metadata, CancellationToken cancellationToken = default);
```

Comment:
I think this counts as a breaking change.

Reply:
The IMessageBatchPublisher.cs is a new interface(created on this PR), also I guess the optional parameter is a syntax sugar